MAUI/LinearGauge/getting-started.md

```diff
@@ -0,0 +1,231 @@
+
+* Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add SfLinearGauge to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.Maui.Gauges] and then install it.
+
+* Import the control namespace `Syncfusion.Maui.Gauges` in XAML or C# code.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+xmlns:gauge="clr-namespace:Syncfusion.Maui.Gauges;assembly=Syncfusion.Maui.Gauges"
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.Gauges;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+* Initialize the SfLinearGauge control
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<gauge:SfLinearGauge />
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Register the handler
+
+Syncfusion.Maui.Core nuget is a dependent package for all Syncfusion controls of .NET MAUI. In the MauiProgram.cs file register the handler for Syncfusion core.
+
+{% highlight c# tabtitle="~/MauiProgram.cs" hl_lines="17" %}
+
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+using Microsoft.Maui.Controls.Compatibility;
+using Microsoft.Maui.Controls.Hosting;
+using Microsoft.Maui.Controls.Xaml;
+using Syncfusion.Maui.Core.Hosting;
+
+namespace GaugeMauiSample
+{
+    public static class MauiProgram
+    {
+        public static MauiApp CreateMauiApp()
+        {
+            var builder = MauiApp.CreateBuilder();
+            builder
+            .UseMauiApp<App>()
+            .ConfigureSyncfusionCore()
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("OpenSans-Regular.ttf", "OpenSansRegular");
+            });
+
+            return builder.Build();
+        }
+    }
+}
+
+{% endhighlight %} 
+
+## Add range to the linear gauge
+
+Ranges contains a list of range elements, to which you can add any number of ranges inside the scale. You can specify the start value, end value, and background color for range using the [`StartValue`](), [`EndValue`](), and [`Fill`]() properties as demonstrated in the following code sample.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<gauge:SfLinearGauge>
+                <gauge:SfLinearGauge.Ranges>
+                    <gauge:LinearRange/>
+                </gauge:SfLinearGauge.Ranges>
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+ SfLinearGauge gauge = new SfLinearGauge();
+        LinearRange range = new LinearRange();
+        gauge.Ranges.Add(range);
+        this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Linear Gauge with Range](images/getting-started/maui-linear-gauge-with-range.PNG)
+
+## Add pointer to the linear gauge
+
+Pointers contains a list of pointer elements, where you can add any number of gauge pointers such as [`BarPointer`](), [`ShapePointer`]() and [`ContentPointer`]() inside the scale to indicate the value.
```

Comment:
Pointers contain a list of pointer elements, in which you can add any number of gauge pointers such as [`BarPointer`](), [`ShapePointer`](), and [`ContentPointer`]() inside the scale to indicate the value.